rollup.config.dist.js

```diff
@@ -1,15 +1,32 @@
-/* global process require */
+/* global process require Set */
 
+import {readFileSync} from 'fs';
 import node from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 
 var pkg = require('./package.json');
 
+var dependencies = pkg => {
+  var deps = Object.keys(pkg.dependencies || {}).concat(Object.keys(pkg.peerDependencies || {}));
+  return Array.from(new Set(deps.concat(deps.flatMap(dependency => (
+    dependencies(require(`${dependency}/package.json`))
+  )))));
```

Comment:
Should this array be sorted in some way?

Reply:
I don't see why. But I'm considering listing them out myself instead of generating them like this, in which case I can choose any order I'd like.

Reply:
I don't like to rely on the internal ordering of unordered data structures. Even if the ordering is deterministic, it may appear arbitrary. Additionally, the relative ordering of elements may change when an element is added to the set, which could result in noisy diffs.

I don't know whether these concerns apply in this particular situation.

Reply:
I don't think it matters for the licenses to be listed in arbitrary order.